sirepo/template/radia.py

```diff
@@ -418,15 +449,45 @@ def _read_or_generate(geom_id, data):
         return get_application_data(data)
 
 
-def _save_fm_sdds(name, f_data, file_path):
+def _read_solution(sim_id):
+    try:
+        return _read_path(sim_id, 'solution')
+    except KeyError:
+        # not solved yet
+        return []
+
+
+def _rot_flat_list(l, r):
```

Comment:
`rot` doesn't mean anything to me. `l` and `r` don't mean anything. function args should be long enough to be readable. Internal variables to a function can be single letter.